flipperdevices/flipperzero-firmware
# [FL-3211][FL-3212] Debug apps: speaker, uart_echo with baudrate

# What's new

- Debug apps: speaker, uart_echo with baudrate

# Verification 

- Check speaker_debug via CLI: `loader open /ext/apps/Debug/speaker_debug.fap Auld L S:d=4,o=6,b=101:g5,c,8c,c,e,d,8c,d,8e,8d,c,8c,e,g,2a,a,g,8e,e,c,d,8c,d,8e,8d,c,8a5,a5,g5,2c`, check sound and then `speaker_debug stop`
- Check uart_echo via CLI: `loader open /ext/apps/Debug/uart_echo.fap`

# Checklist (For Reviewer)
